<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PacijentLoginWindow</class>
 <widget class="QMainWindow" name="PacijentLoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ambulanta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="pacijentLoginSIfra">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>146</y>
      <width>72</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Å ifra:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pacijentLoginLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>100</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pacijent</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pacijent_login">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>270</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Uloguj se</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pacijent_registruj">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>160</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registruj se</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_pacijent_login">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>97</y>
      <width>331</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="pacijentLoginIme">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>60</y>
      <width>42</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sifra_pacijent_login">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>189</y>
      <width>331</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>220</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
